fmc/src/flow/rt_alias.rs

```diff
@@ -21,11 +21,18 @@ use crate::fmc_env::FmcEnv;
 use crate::HandOff;
 use caliptra_common::cprintln;
 use caliptra_common::crypto::Ecc384KeyPair;
+use caliptra_common::HexBytes;
 use caliptra_drivers::{
     okref, CaliptraError, CaliptraResult, Ecc384PubKey, Hmac384Data, Hmac384Key, KeyId, KeyReadArgs,
 };
+use caliptra_x509::{NotAfter, NotBefore, RtAliasCertTbs, RtAliasCertTbsParams};
+
 const SHA384_HASH_SIZE: usize = 48;
 
+extern "C" {
+    static mut RTALIAS_TBS_ORG: u8;
+}
```

Comment:
```suggestion
const RT_ALIAS_SIZE = 0x1000;
extern "C" {
    static mut RTALIAS_TBS_ORG: [u8; RT_ALIAS_SIZE];
}
```

Reply:
This allows you to refer to the slice with `let cert_region = unsafe { &RTALIAS_TBS_ORG[..] };`. This makes bounds-checking easier/less error prone.